### ISSUE TITLE
Chat room

### ISSUE THREAD

**Issue by ghost** [zhanzhenzhen/futurescript#10]
What about a chat room about futurescript (in English)? It's easy to create a Gitter chat and I think it's useful.
Thank you.

**Comment by zhanzhenzhen** [comment]
Do other languages (e.g. CoffeeScript) have a chat room? Is it useful? I think GitHub issues is already a good solution.

**Comment by ghost** [comment]
CoffeeScript has a Gitter chat room, as well as LiveScript , TypeScript and many libraries. 
I think it's useful. It's real-time, so it's more... fluent than sending a issue, waiting a response, writing your response, waiting again, etc. It's a good complement to issues. And to create a Gitter chat room is a snap.

**Comment by zhanzhenzhen** [comment]
Ok, when it gets popular I will consider it. Now it seems useless because I guess there will be not enough people in the room. Online people are always fewer than offline.

**Comment by kristianmandrup** [comment]
Yes, Create a Slack or Gitter channel even with few ppl, is great for asking questions. Better than Issues which should be related to bugs/features, not general questions. Cheers!
